go.mod

```diff
@@ -1,12 +1,12 @@
 module github.com/textileio/go-tableland
 
-go 1.18
+go 1.19
 
 require (
 	cloud.google.com/go/bigquery v1.45.0
 	cloud.google.com/go/logging v1.6.1
 	github.com/XSAM/otelsql v0.20.0
-	github.com/ethereum/go-ethereum v1.10.26
+	github.com/ethereum/go-ethereum v1.11.3
```

Comment:
go-ethereum v1.11.3 drops the support for go v1.18; hence we are updating to go 1.19.